packages/core/src/baseclient.ts

```diff
@@ -507,6 +507,7 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
     if (event.contexts && event.contexts.trace) {
       normalized.contexts = {};
       normalized.contexts.trace = event.contexts.trace;
+      normalized.contexts.baggage = event.contexts.baggage;
```

Comment:
I wonder if line 508 is a problem. I thought that contexts could hold arbitrary data 🤔

Reply:
Yup contexts can hold arbitrary data!

Reply:
So this line shouldn't exist, right? I checked and it was added in #5171 to normalize trace context data but IIUC what's going on here, we're accidentally erasing all other contexts except the ones we're adding back (which is why I added line 510).

Reply:
Removed the line for the time being in eebeb6b53f0b1daafbd26c160bf4414d56f54da9 (happy to revisit this if it's a problem). I checked, the other elements in normalized.contexts are normalized a few lines above: https://github.com/getsentry/sentry-javascript/blob/eebeb6b53f0b1daafbd26c160bf4414d56f54da9/packages/core/src/baseclient.ts#L492-L494